packages/admin/cms-admin/src/index.ts

```diff
@@ -56,6 +56,7 @@ export { useDamAcceptedMimeTypes } from "./dam/config/useDamAcceptedMimeTypes";
 export { useDamConfig } from "./dam/config/useDamConfig";
 export { useCurrentDamFolder } from "./dam/CurrentDamFolderProvider";
 export { DamPage } from "./dam/DamPage";
+export type { FileWithCustomMetaData } from "./dam/DataGrid/fileUpload/useDamFileUpload";
```

Comment:
I'm not sure about the name if we export this. Maybe `FileUploadWithCustomMetadata`?

What do you think @johnnyomair?

Reply:
Maybe `DamFileUpload` since it's used in the `useDamFileUpload` hook?

Reply:
Or `FileWithDamUploadMetadata`? I kind of like that

Reply:
@VP-DS please rename the type.

Reply:
a9ae60872bde2f48ae7d0d8ca93d5c4bd7f530dc